src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
```

Comment:
The destructuring assignment is correct, but it's a good practice to check if the `declaration` is not an empty string before splitting it to avoid creating empty `property` and `value` variables.